includes/virtual-machines-common-expand-os-disk.md

```diff
@@ -8,6 +8,9 @@ When you create a new virtual machine (VM) in a Resource Group by deploying an i
 > Azure has two different deployment models for creating and working with resources: Resource Manager and Classic. This article covers using the Resource Manager model. Microsoft recommends that most new deployments use the Resource Manager model.
 > 
 > 
+> [!WARNIG]
+> Resizing Azure Machine will cause the machine to restart
```

Comment:
Reword the sentence: Resizing the OS Disk of an Azure Virtual Machine will cause it to restart.

Reply:
Concur with bahreex.